index.js

```diff
@@ -27,7 +27,7 @@ function download (version, callback) {
   var tarball = path.join(tmpdir, `electron-${version}.tgz`)
 
   var extractor = tar.extract(tmpdir, {
-    ignore: (name) => { return !name.match('docs/')} }
+    ignore: (name) => { return !name.match('docs' + (process.platform === 'win32' ? '\\\\' : '/'))} }
```

Comment:
What about `/[\\/]docs[\\/]/`?

That's how electron-is-dev does it: https://github.com/sindresorhus/electron-is-dev/blob/master/index.js